stock_reserve/model/__init__.py

```diff
@@ -19,5 +19,7 @@
 #
 ##############################################################################
 
+from . import stock_location
```

Comment:
Please sort in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports